### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by damiisdandy** [pariola/paystack#17]
Would be really nice if you had Typescript support

**Comment by pariola** [comment]
This is now in progress in branch [feat/typings](https://github.com/pariola/paystack/tree/feat/typings)

**Comment by a-k-o-r-e-d-e** [comment]
Hello @pariola, please what the update on the Typescript support. 

Do you need assistance with any part, I am willing to contribute.

**Comment by pariola** [comment]
@Lone-Wolf17 Yes that would be very much appreciated. I think my approach would be to first support the latest API from Paystack (started working on this in [feat/api-updates](https://github.com/pariola/paystack/tree/feat/api-updates)) before continuing Typescript support ([feat/typings](https://github.com/pariola/paystack/tree/feat/typings)).

**Comment by a-k-o-r-e-d-e** [comment]
can you please create an issue with a checklist of the API updates so one can know what has been done and what next to jump on. 

please see this link https://github.com/pintariching/rustle/issues/7 for an example of such a checklist.

**Comment by wjbright** [comment]
Hi @pariola @Lone-Wolf17 , checking in on this. Do you need support on this PR? Would be happy to contribute.

**Comment by gray-adeyi** [comment]
Hi! try out this package. It's not yet a major release but it supports typescript and other JavaScript runtime like bun and Deno https://www.npmjs.com/package/@gray-adeyi/paystack-sdk

**Comment by gray-adeyi** [comment]
@gray-adeyi/paystack-sdk 0.2.4 release is now available with even better type support 🔥 https://www.npmjs.com/package/@gray-adeyi/paystack-sdk
